tgui/packages/tgui/interfaces/Hypospray.tsx

```diff
@@ -0,0 +1,66 @@
+import { BooleanLike } from 'common/react';
+
+import { useBackend } from '../backend';
+import { Box, Button } from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  InjectMode: BooleanLike;
+  CurrentLabel: string;
+  CurrentTag: string;
+  TransferAmount: number;
+  IsAdvanced: boolean;
+};
+
+export const Hypospray = (props) => {
+  const { act, data } = useBackend<Data>();
+  const { InjectMode, CurrentLabel, CurrentTag, TransferAmount, IsAdvanced } =
+    data;
+  return (
+    <Window width={300} height={375}>
+      <Box>
```

Comment:
sidenote: this probably should just be a <Stack> or <Labelledlist> but this is mergable if reviews get adressed